src/Core/Grid/Filter/FilterCollection.php

```diff
@@ -58,6 +58,20 @@ public function remove($filterName)
         return $this;
     }
 
+    /**
+     * @param string $filterName
+     *
+     * @return FilterInterface|null return null if no filter with given filter name
+     */
+    public function get($filterName)
+    {
+        if (isset($this->filters[$filterName])) {
+            return $this->filters[$filterName];
+        }
+
+        return null;
```

Comment:
```suggestion
        return $this->filters[$filterName] ?? null;
```

Reply:
The null coalescing operator is a new PHP short operator that kindof serves the same purpose that the ternary operator, which is to avoid heavy `if/else` statements.

However the use of these short operators has been debated for a long time (and will probably continue for years) because it might hurt the code readability.

Stackoverflow is full of fiery debates about whether or not to use ternary operators, with both sides having both passionate and logical arguments 😄 (see [this one](https://stackoverflow.com/questions/694814/ternary-operator-bad-or-good-practice) or [this one](https://softwareengineering.stackexchange.com/questions/28314/ternary-operator-considered-harmful)).

I personally side with the people suggesting not to use it, except for very obvious usecases. I like [this position](https://pressupinc.com/blog/2015/03/ternary-operators-considered-harmful/): _"If conciseness is more valuable, I should use a single-line ternary operation. If clarity is more valuable, I should use the English keywords of if and else."_

Considering that PrestaShop is a wide and complex project which can be used by both junior and senior developers, I suggest we keep the if/else structure as it's the most common and widely known, and we keep the maximum readability (with the cost of a few more lines indeed) 🤔